.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+
+updates:
+  # Update npm packages
+  - package-ecosystem: npm
+    directory: /
+
+    # Bump package.json versions too
+    # in addition to security updates
+    allow:
+      - dependency-type: direct
+    versioning-strategy: increase
```

Comment:
This line is necessary to save bumped semver ranges to `package.json`

Otherwise Dependabot only writes to `package-lock.json`

Reply:
This seems good, though I feel if this is worth a comment on the PR then it should be a comment in the code

Reply:
I've called out `package.json` on line 8 already, think I need to mention the default `package-lock.json` behaviour too?

Reply:
Ah sorry my bad